internal/core/items/text.rs

```diff
@@ -275,6 +275,7 @@ pub struct TextInput {
     pub pressed: core::cell::Cell<bool>,
     pub single_line: Property<bool>,
     pub cached_rendering_data: CachedRenderingData,
+    preferred_x_pos: core::cell::Cell<f32>,
```

Comment:
Please add a comment:
```suggestion
    /// This is the anchor x position used when doing cursor navigation
    preferred_x_pos: core::cell::Cell<f32>,
```

Reply:
The word anchor is already used for selecting text. But I'll add another comment.